### DISCUSSION TITLE
CommandList error

### DISCUSSION THREAD

**Discussion by luonganhtu19** [dentsusoken/iPLAss#1357] [General]
My project using xml 
`
<service>
<interfaceName>jp.co.isid.mtp.impl.metadata.MetaDataRepository</interfaceName>

		
		
		<property name="resourcePath" value="xxx.xml" additional="true" />
		

		
		
		<property name="annotatedClass" value="xx.CommandList" additional="true" />
		<property name="annotatedClass" value="xx.CommandList" additional="true" />
<service>`
Url of link CommandList is correct.  I using in xml for build file war but my web iplass and display server 500 (server error).

**Comment by HiguchiKiyoshi** [comment]
Hi @luonganhtu19 ,

Thanks for using iPLAss.

The 500 error could be due to a variety of reasons, and I believe the logs are output on the server side, so please look into the cause.

Also, in the question you gave me, the package name is "jp.co.isid.mtp..." and I think it is a setting of iPLAss ver. 2.x.
Is the error occurring now when you run iPLAss ver. 3.x?
If so, the error may be due to an old configuration file.

Or, if the error is occurring with iPLAss ver. 2.x, it is a version before the OSS release, so please ask via the paid product support associated with the project.

**Comment by luonganhtu19** [comment]
Detail log. I don't no why error. Coludyou assist me why the error?
`
    "log": "11-Dec-2023 13:53:30.171 WARNING [main] org.apache.tomcat.util.digester.SetPropertiesRule.begin [SetPropertiesRule]{Server/Service/Engine/Host/Valve} Setting property 'resolveHosts' to 'false' did not find a matching property."
}
{
    "log": "11-Dec-2023 13:53:30.175 INFO [main] org.apache.catalina.core.AprLifecycleListener.lifecycleEvent Loaded Apache Tomcat Native library [1.2.25] using APR version [1.6.3]."
}
{
    "log": "11-Dec-2023 13:53:30.176 INFO [main] org.apache.catalina.core.AprLifecycleListener.lifecycleEvent APR capabilities: IPv6 [true], sendfile [true], accept filters [false], random [true]."
}
{
    "log": "11-Dec-2023 13:53:30.176 INFO [main] org.apache.catalina.core.AprLifecycleListener.lifecycleEvent APR/OpenSSL configuration: useAprConnector [false], useOpenSSL [true]"
}
{
    "log": "11-Dec-2023 13:53:30.179 INFO [main] org.apache.catalina.core.AprLifecycleListener.initializeSSL OpenSSL successfully initialized [OpenSSL 1.0.2k-fips  26 Jan 2017]"
}
{
    "log": "11-Dec-2023 13:53:30.459 INFO [main] org.apache.coyote.AbstractProtocol.init Initializing ProtocolHandler [\"http-nio-8080\"]"
}
{
    "log": "11-Dec-2023 13:53:30.471 INFO [main] org.apache.tomcat.util.net.NioSelectorPool.getSharedSelector Using a shared selector for servlet write/read"
}
{
    "log": "11-Dec-2023 13:53:30.557 INFO [main] org.apache.coyote.AbstractProtocol.init Initializing ProtocolHandler [\"ajp-nio-127.0.0.1-8091\"]"
}
{
    "log": "11-Dec-2023 13:53:30.559 INFO [main] org.apache.tomcat.util.net.NioSelectorPool.getSharedSelector Using a shared selector for servlet write/read"
}
{
    "log": "11-Dec-2023 13:53:30.560 INFO [main] org.apache.catalina.startup.Catalina.load Initialization processed in 2591 ms"
}
{
    "log": "11-Dec-2023 13:53:30.857 INFO [main] org.apache.catalina.core.StandardService.startInternal Starting service [Catalina]"
}
{
    "log": "11-Dec-2023 13:53:30.858 INFO [main] org.apache.catalina.core.StandardEngine.startInternal Starting Servlet Engine: Apache Tomcat/8.5.59"
}
{
    "log": "11-Dec-2023 13:53:30.957 INFO [localhost-startStop-1] org.apache.catalina.startup.HostConfig.deployWAR Deploying web application archive [/usr/local/tomcat/webapps/mngcheck.war]"
}
{
    "log": "11-Dec-2023 13:53:32.663 INFO [localhost-startStop-1] org.apache.jasper.servlet.TldScanner.scanJars At least one JAR was scanned for TLDs yet contained no TLDs. Enable debug logging for this logger for a complete list of JARs that were scanned but no TLDs were found in them. Skipping unneeded JARs during scanning can improve startup time and JSP compilation time."
}
{
    "log": "11-Dec-2023 13:53:32.865 INFO [localhost-startStop-1] org.apache.catalina.startup.HostConfig.deployWAR Deployment of web application archive [/usr/local/tomcat/webapps/mngcheck.war] has finished in [1,907] ms"
}
{
    "log": "11-Dec-2023 13:53:32.866 INFO [localhost-startStop-1] org.apache.catalina.startup.HostConfig.deployWAR Deploying web application archive [/usr/local/tomcat/webapps/ROOT.war]"
}
{
    "log": "11-Dec-2023 13:53:38.676 INFO [localhost-startStop-1] org.apache.jasper.servlet.TldScanner.scanJars At least one JAR was scanned for TLDs yet contained no TLDs. Enable debug logging for this logger for a complete list of JARs that were scanned but no TLDs were found in them. Skipping unneeded JARs during scanning can improve startup time and JSP compilation time."
}
{
    "log": "11-Dec-2023 13:53:41.272 INFO [localhost-startStop-1] com.sun.xml.ws.transport.http.servlet.WSServletContextListener.contextInitialized WSSERVLET12: JAX-WS context listener initializing"
}
{
    "log": "11-Dec-2023 13:53:47.171 INFO [localhost-startStop-1] com.sun.xml.ws.server.MonitorBase.createRoot Metro monitoring rootname successfully set to: com.sun.metro:pp=/,type=WSEndpoint,name=-CommandInvoker-CommandInvoker"
}
{
    "log": "11-Dec-2023 13:53:49.463 INFO [localhost-startStop-1] com.sun.xml.ws.server.MonitorBase.createRoot Metro monitoring rootname successfully set to: com.sun.metro:pp=/,type=WSEndpoint,name=-DefinitionManager-DefinitionManager"
}
{
    "log": "11-Dec-2023 13:53:49.858 INFO [localhost-startStop-1] com.sun.xml.ws.server.MonitorBase.createRoot Metro monitoring rootname successfully set to: com.sun.metro:pp=/,type=WSEndpoint,name=-BinaryOperation-BinaryOperation"
}
{
    "log": "11-Dec-2023 13:53:49.869 INFO [localhost-startStop-1] com.sun.xml.ws.transport.http.servlet.WSServletDelegate.<init> WSSERVLET14: JAX-WS servlet initializing"
}
{
    "log": "11-Dec-2023 13:53:49.870 SEVERE [localhost-startStop-1] org.apache.tomcat.util.descriptor.web.SecurityConstraint.findUncoveredHttpMethods For security constraints with URL pattern [/*] only the HTTP methods [TRACE HEAD OPTIONS] are covered. All other methods are uncovered."
}
{
    "log": "11-Dec-2023 13:53:50.165 INFO [localhost-startStop-1] com.sun.jersey.server.impl.application.WebApplicationImpl._initiate Initiating Jersey application, version 'Jersey: 1.19.1 03/11/2016 02:08 PM'"
}
{
    "log": "11-Dec-2023 13:53:50.272 INFO [localhost-startStop-1] com.sun.jersey.server.impl.application.DeferredResourceConfig$ApplicationHolder.<init> Instantiated the Application class jp.co.isid.mtp.impl.webapi.rs.MtpRestApplication"
}
{
    "log": "11-Dec-2023 13:53:51.270 WARNING [localhost-startStop-1] com.sun.jersey.spi.inject.Errors.processErrorMessages The following warnings have been detected with resource and/or provider classes:"
}
{
    "log": "  WARNING: A sub-resource method, public javax.ws.rs.core.Response jp.co.isid.mtp.impl.webapi.rs.RestBinaryManager.doUpload(javax.ws.rs.core.Request,javax.servlet.http.HttpServletRequest,javax.servlet.ServletContext,javax.servlet.http.HttpServletResponse), with URI template, \"/\", is treated as a resource method"
}
{
    "log": "  WARNING: A sub-resource method, public javax.ws.rs.core.Response jp.co.isid.mtp.impl.webapi.rs.RestBinaryManager.preflightUpload(javax.servlet.ServletContext,javax.servlet.http.HttpServletRequest,javax.servlet.http.HttpServletResponse), with URI template, \"/\", is treated as a resource method"
}
{
    "log": "11-Dec-2023 13:53:51.356 INFO [localhost-startStop-1] org.apache.catalina.startup.HostConfig.deployWAR Deployment of web application archive [/usr/local/tomcat/webapps/ROOT.war] has finished in [18,490] ms"
}
{
    "log": "11-Dec-2023 13:53:51.360 INFO [main] org.apache.coyote.AbstractProtocol.start Starting ProtocolHandler [\"http-nio-8080\"]"
}
{
    "log": "11-Dec-2023 13:53:51.371 INFO [main] org.apache.coyote.AbstractProtocol.start Starting ProtocolHandler [\"ajp-nio-127.0.0.1-8091\"]"
}
{
    "log": "11-Dec-2023 13:53:51.686 INFO [main] org.apache.catalina.startup.Catalina.start Server startup in 21125 ms"
}
{
    "log": "11-Dec-2023 13:54:12.760 SEVERE [ajp-nio-127.0.0.1-8091-exec-8] org.apache.catalina.core.StandardWrapperValve.invoke Servlet.service() for servlet [default] in context with path [] threw exception"
}
{
    "log": "\tjava.lang.NullPointerException"
}
{
    "log": "\t\tat jp.co.isid.mtp.impl.web.TenantFilter.doFilter(TenantFilter.java:141)"
}
{
    "log": "\t\tat org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)"
}
{
    "log": "\t\tat org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)"
}
{
    "log": "\t\tat org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:199)"
}
{
    "log": "\t\tat org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)"
}
{
    "log": "\t\tat org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:544)"
}
{
    "log": "\t\tat org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)"
}
{
    "log": "\t\tat org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81)"
}
{
    "log": "\t\tat org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)"
}
{
    "log": "\t\tat org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)"
}
{
    "log": "\t\tat org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)"
}
{
    "log": "\t\tat org.apache.coyote.ajp.AjpProcessor.service(AjpProcessor.java:525)"
}
{
    "log": "\t\tat org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)"
}
{
    "log": "\t\tat org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:818)"
}
{
    "log": "\t\tat org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1626)"
}
{
    "log": "\t\tat org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)"
}
{
    "log": "\t\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)"
}
{
    "log": "\t\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)"
}
{
    "log": "\t\tat org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)"
}
{
    "log": "\t\tat java.base/java.lang.Thread.run(Thread.java:834)"
}
{
    "log": "11-Dec-2023 13:59:02.762 INFO [main] org.apache.catalina.core.StandardServer.await A valid shutdown command was received via the shutdown port. Stopping the Server instance."
}
{
    "log": "11-Dec-2023 13:59:02.764 INFO [main] org.apache.coyote.AbstractProtocol.pause Pausing ProtocolHandler [\"http-nio-8080\"]"
}
{
    "log": "11-Dec-2023 13:59:02.770 INFO [main] org.apache.coyote.AbstractProtocol.pause Pausing ProtocolHandler [\"ajp-nio-127.0.0.1-8091\"]"
}
{
    "log": "11-Dec-2023 13:59:02.775 INFO [main] org.apache.catalina.core.StandardService.stopInternal Stopping service [Catalina]"
}
{
    "log": "11-Dec-2023 13:59:02.788 INFO [localhost-startStop-2] com.sun.xml.ws.transport.http.servlet.WSServletDelegate.destroy WSSERVLET15: JAX-WS servlet destroyed"
}
{
    "log": "11-Dec-2023 13:59:02.960 INFO [localhost-startStop-2] com.sun.xml.ws.server.WSEndpointImpl.closeManagedObjectManager Closing Metro monitoring root: com.sun.metro:pp=/,type=WSEndpoint,name=-CommandInvoker-CommandInvoker"
}
{
    "log": "11-Dec-2023 13:59:02.963 INFO [localhost-startStop-2] com.sun.xml.ws.server.WSEndpointImpl.closeManagedObjectManager Closing Metro monitoring root: com.sun.metro:pp=/,type=WSEndpoint,name=-DefinitionManager-DefinitionManager"
}
{
    "log": "11-Dec-2023 13:59:02.963 INFO [localhost-startStop-2] com.sun.xml.ws.server.WSEndpointImpl.closeManagedObjectManager Closing Metro monitoring root: com.sun.metro:pp=/,type=WSEndpoint,name=-BinaryOperation-BinaryOperation"
}
{
    "log": "11-Dec-2023 13:59:02.964 INFO [localhost-startStop-2] com.sun.xml.ws.transport.http.servlet.WSServletContextListener.contextDestroyed WSSERVLET13: JAX-WS context listener destroyed"
}
{
    "log": "11-Dec-2023 13:59:03.161 INFO [main] org.apache.coyote.AbstractProtocol.stop Stopping ProtocolHandler [\"http-nio-8080\"]"
}
{
    "log": "11-Dec-2023 13:59:03.163 INFO [main] org.apache.coyote.AbstractProtocol.stop Stopping ProtocolHandler [\"ajp-nio-127.0.0.1-8091\"]"
}
{
    "log": "11-Dec-2023 13:59:03.166 INFO [main] org.apache.coyote.AbstractProtocol.destroy Destroying ProtocolHandler [\"http-nio-8080\"]"
}
{
    "log": "11-Dec-2023 13:59:03.167 INFO [main] org.apache.coyote.AbstractProtocol.destroy Destroying ProtocolHandler [\"ajp-nio-127.0.0.1-8091\"]"`
    
![image](https://github.com/ISID/iPLAss/assets/47765029/d09f12fc-292a-4af1-9110-6c45d6e7cb50)

**Reply by HiguchiKiyoshi** [reply]
Hi @luonganhtu19 ,

From the log, this is for iPLAss ver2.x.
Since ver2.x is not an OSS version, and since it is not good to disclose your project information to the public, please contact us through the paid support associated with your project.
